irods/exception.py

```diff
@@ -67,8 +69,60 @@ def __init__(self, name, bases, attrs):
             iRODSExceptionMeta.codes[attrs['code']] = self
 
 
+class Errno:
```

Comment:
Is it safe to use the name, `Errno`?
Can it lead to name collisions?

Reply:
I think we can stay with "Errno".  If there were collisions in the future, they would affect only this module and it could be corrected pretty easily.